lib/activerecord-import/active_record/adapters/postgresql_adapter.rb

```diff
@@ -3,5 +3,15 @@
 
 class ActiveRecord::ConnectionAdapters::PostgreSQLAdapter
   include ActiveRecord::Import::PostgreSQLAdapter
+
+  alias_method :post_sql_statements_orig, :post_sql_statements
+  def post_sql_statements( table_name, options ) # :nodoc:
+    unless options[:primary_key].blank?
+      post_sql_statements_orig(table_name, options) << (" RETURNING #{options[:primary_key]}")
+    else
+      post_sql_statements_orig(table_name, options)
+    end
+  end
```

Comment:
Replaced `alias_method` with use of module/super: https://github.com/zdennis/activerecord-import/commit/331004696d9ae29aafe91925e0861b278a949bf8